src/css/custom.css

```diff
@@ -41,6 +41,10 @@
   margin-top: -60px;
 }
 
+section {
+  padding: 100px 0px !important;
+}
```

Comment:
1. Why do we override it here with `!important` instead of fixing it in `src/components/SectionLayout/Section.tsx`?
2. Do we want 100% padding for mobiles too?

Reply:
1. 🙈 I change it and fix it in `src/components/SectionLayout/Section.tsx` like you suggested 
2. Firstly I thought that yes, but now when I am looking at this once again maybe it is better to make it a little smaller for mobiles - I change it to 75px for them :D